### PR TITLE
[improvement](statistics)Skip auto analyze when table row count is not fully reported. (return -1)

### PR DESCRIPTION
**Pull request by Jibing-Li**
Skip auto analyze when table row count is not fully reported. Not fully reported means row count is -1.

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run p0

**Comment by Jibing-Li** [comment]
run p0

**Comment by Jibing-Li** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java [hunk 1]
```diff
@@ -203,6 +206,13 @@ protected AnalysisInfo createAnalyzeJobForTbl(
         if (StatisticsUtil.enablePartitionAnalyze() && table.isPartitionedTable()) {
             analysisMethod = AnalysisMethod.FULL;
         }
+        if (table instanceof OlapTable && analysisMethod.equals(AnalysisMethod.SAMPLE)) {
+            OlapTable ot = (OlapTable) table;
+            if (ot.getRowCountForIndex(ot.getBaseIndexId(), true) == -1) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java:211]: -1 is a magic number, could we use a static variable instead?

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run p0

**Review by morrySnow — APPROVED** [review]

**Review by zfr9527 — APPROVED** [review]
LGTM

**Pull request merged**